lib/mocaccino.js

```diff
@@ -13,6 +13,7 @@ var through = require('through2');
 var resolve = require('resolve');
 var listen  = require('listen');
 var supportsColor = require('supports-color');
+var mochaReporters = require('mocha').reporters;
```

Comment:
Ha! This was already an issue before. `mocha` is required by the node runner. I guess it just happens to work because one usually already has a mocha dependency top level.